src/browser/documentation/guides/intro.tsx

```diff
@@ -55,26 +55,45 @@ const slides = [
     </div>
     <div className="col-sm-5">
       <p>
-        The editor is the primary interface for entering and running commands.
-        Enter Cypher queries to work with graph data. Use client-side commands
-        like
-        <code>:help</code> for other operations.
+        Editor pane is used to edit and run for Cypher statements and Browser
+        commands. Browser commands begin with <code>:</code>, for example{' '}
+        <code>:help</code>
+      </p>
+      <table>
+        <tr>
```

Comment:
The table tag should still be present, tbody should be inside of table instead of replacing it. The structure here should be
```
<table>
  <tbody>
    <tr>
      <td></td>
    </tr>
  </tbody>
</table>
```